docs/source/activelearning.rst

```diff
@@ -2,3 +2,27 @@ Active Learning
 ~~~~~~~~~~~~~~~
 
 Introduction of Active Learning on (academic) literature.
+
+Active learning denotes the scenario in which a reviewer actively provides classifications for references which are presented by the model [3,4]. Subsequently, the model retrains based on the new input. When it has retrained, it decides on a new reference to be annotated and it shows this to the reviewer. In this way, the annotated dataset starts out small and iteratively grows in size [4]. In the case of Automated Systematic Review, this process is necessary to provide some initial classifications to the model, because the reviewer starts with a dataset without annotations.
```

Comment:
I think that the first sentences could be a bit more clear. If I may, here is a suggestion: 

> Active learning denotes the scenario in which the reviewer is labeling references that are presented by a machine learning model. The machine learning model learns from the reviewers labeling decisions and uses this knowledge in selecting the reference that will be presented to the reviewer next.

Reply:
So replace the sentences "active learning denotes... to the reviewer" with your suggestion? Sounds like a good improvement to me!